patches/minecraft/net/minecraft/world/level/levelgen/feature/treedecorators/AlterGroundDecorator.java.patch

```diff
@@ -0,0 +1,62 @@
+--- a/net/minecraft/world/level/levelgen/feature/treedecorators/AlterGroundDecorator.java
++++ b/net/minecraft/world/level/levelgen/feature/treedecorators/AlterGroundDecorator.java
+@@ -35,21 +_,22 @@
+       }
+ 
+       if (!list.isEmpty()) {
++         var eventProvider = net.minecraftforge.event.ForgeEventFactory.alterGround(p_225969_, list, this.f_69303_::m_213972_);
```

Comment:
I think we can shelve that for future development of the event if there is a usecase for messing with the y-level, personally it wouldn't make any sense (the y-level is the detected ground level, so messing with it would just result in transforming something that isn't the ground).

I imagine most people will not need to care what the y-level is, as they will just be supplying a `StateProvider` which will only be invoked on positions matching the correct y-level.

Reply:
Fair enough, especially given the change/addition of passing the y value can always be made in a non-breaking way if someone down the road ends up needing it.